index.js

```diff
@@ -68,12 +74,15 @@ function auto (client) {
 
       pipeline.queued++
 
```

Comment:
I think it always add a catch handler.

Reply:
Are you saying ioredis always adds a catch handler or are you saying we should?

Reply:
I _think_ ioredis always add a catch handler in its promises because we do not have an unhandledRejection when the callback is called with an error and the promise rejected. Another option is to _not_ call `reject(err)` if there is a callback, but rather `resolve()`, or even nothing